Tasks/SqlAzureDacpacDeploymentV1/SqlAzureActions.ps1

```diff
@@ -328,50 +345,45 @@ function Get-AgentIPRange
 
     [hashtable] $IPRange = @{}
 
-    if (Get-Command -Name "Invoke-Sqlcmd" -ErrorAction SilentlyContinue)
-    {
-      try {
-        Write-Verbose "Reaching SqlServer to check connection by running Invoke-SqlCmd"
-        Write-Verbose "Run-InlineSql -authenticationType $authenticationType -serverName $serverName -databaseName $databaseName -sqlUserName $sqlUserName -sqlPassword $sqlPassword -sqlInline `"select getdate()`" -connectionString $connectionString -ErrorVariable errors -ConnectionTimeout 120 | Out-String"
-
-        $output = Run-InlineSql -authenticationType $authenticationType -serverName $serverName -databaseName $databaseName -sqlUserName $sqlUserName -sqlPassword $sqlPassword -sqlInline "select getdate()" -connectionString $connectionString -ErrorVariable errors | Out-String
-      }
-      catch {
-        Write-Verbose "Failed to reach SQL server $serverName. $($_.Exception.Message)"
-      }
+    if (Get-Command -Name "Invoke-Sqlcmd" -ErrorAction SilentlyContinue) {
+        try {
+            Write-Verbose "Reaching SqlServer to check connection by running Invoke-SqlCmd"
+            Write-Verbose "Run-InlineSql -authenticationType $authenticationType -serverName $serverName -databaseName $databaseName -sqlUserName $sqlUserName -sqlPassword $sqlPassword -sqlInline `"select getdate()`" -connectionString $connectionString -ErrorVariable errors -ConnectionTimeout 120 | Out-String"
```

Comment:
Need to add token here as well. Otherwise it will fail at this step

Reply:
added where needed